code/modules/mob/living/carbon/human/human.dm

```diff
@@ -455,30 +455,34 @@
 			return FALSE
 
 		if (target.stat == DEAD || HAS_TRAIT(target, TRAIT_FAKEDEATH))
-			to_chat(src, span_warning("[target.name] is dead!"))
+			balloon_alert(src, "[target.name] is dead!")
```

Comment:
```suggestion
			balloon_alert(src, "this person is dead!")
```

if someone is named "YOLO SWAGGINS ABLOGLBOBLBOGLBOOBOBLOOO" (blame admins) then this balloon alert will be very ugly.

Reply:
if you are going that route: 
```suggestion
			balloon_alert(src, "they're dead!")
```

Reply:
Fixed by commit a925ae7c9746caf21aa228cbcf8b9ffb1b5e6dd4 using `p_they()` and `p_are()`